src/ec/packages/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig

```diff
@@ -156,7 +156,7 @@
 {% endif %}
 
 {# Print the result #}
-
+{% if group != 'group3' %}
```

Comment:
Mmhh, despite the fact that the template produces the right html (perfectly) i don't necessarily understand the reason why duplicating so much code, it looks like this "variant" is just a very simplified version of the other two, but it's markup corresponds to the "first part" of the template, so i would maybe use the same. If you did this on purpose considering also this other option don't consider this as a "mandatory" thing to be changed, you can explain the reasons why you chose to do like this.. ;)